tests/state_test_util.go

```diff
@@ -85,18 +85,17 @@ type stInfo struct {
 	LLLCVersion        string                 `json:"lllcversion"`
 	Source             string                 `json:"source"`
 	SourceHash         string                 `json:"sourceHash"`
-	WrittenWith        string                 `json:"written-with"`
-	Parent             string                 `json:"parent"`
-	ParentSha1Sum      string                 `json:"parentSha1Sum"`
```

Comment:
These three fields were specific to core-geth test generation metadata. I've removed them since their information be inferred with version control, and I use the `FilledWith` field to define the core-geth version (instead of `WrittenWith`).

Reply:
Also `Chainspecs` is disused since the tests generation creates the `GeneralStateTests_configs.json` which defines the test configurations; making references to configs by file unnecessary.